googlemock/include/gmock/gmock-actions.h

```diff
@@ -570,13 +570,9 @@ class PolymorphicAction {
 
    private:
     Impl impl_;
-
-    GTEST_DISALLOW_ASSIGN_(MonomorphicImpl);
```

Comment:
`MonomorphicImpl& operator=(const MonomorphicImpl&) = delete;`
The move ctor doesn't seem to matter for this class.